python/ray/rllib/test/test_checkpoint_restore.py

```diff
@@ -26,11 +25,14 @@ def get_mean_action(alg, obs):
 for (cls, default_config) in [
         (DQNAgent, DQN_CONFIG),
         (PPOAgent, PG_CONFIG),
-        # TODO(ekl) this fails with multiple ES instances in a process
+        # https://github.com/ray-project/ray/issues/1062
         # (ESAgent, ES_CONFIG),
-        (A3CAgent, A3C_CONFIG)]:
+        # https://github.com/ray-project/ray/issues/1061
```

Comment:
This should be addressed by #1066; but we can merge this first and then open a new PR to resolve this once the tests pass for 1066.